client/analytics/components/report-filters/index.js

```diff
@@ -14,41 +15,55 @@ import {
 	isoDateFormat,
 } from '@woocommerce/date';
 import { recordEvent } from '@woocommerce/tracks';
+import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import { CurrencyContext } from '../../../lib/currency-context';
+import { STORE_KEY as CES_STORE_KEY } from '../../../customer-effort-score-tracks/data/constants';
 
 class ReportFilters extends Component {
 	constructor() {
 		super();
-		this.trackDateSelect = this.trackDateSelect.bind( this );
-		this.trackFilterSelect = this.trackFilterSelect.bind( this );
-		this.trackAdvancedFilterAction = this.trackAdvancedFilterAction.bind(
-			this
+		this.onDateSelect = this.onDateSelect.bind( this );
+		this.onFilterSelect = this.onFilterSelect.bind( this );
+		this.onAdvancedFilterAction = this.onAdvancedFilterAction.bind( this );
+	}
+
+	showCesSurveyTrack() {
+		const { addCesSurveyTrack } = this.props;
+		addCesSurveyTrack(
+			'woocommerce_admin_analytics_filtered',
```

Comment:
The action name doesn't need the `woocommerce_admin` prefix, `analytics_filtered` should be fine

Reply:
Thank you :) I thought we always need to prefix it for some reason. Updated it with commit [8204aa3](https://github.com/woocommerce/woocommerce-admin/commit/8204aa31cf41f819b56943977a03b9519a345d29#diff-094f070da895dc2eeb07300abbba96a33d495e023ae085eff766a72754ceb281R53Rc)